paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -95,6 +95,10 @@ internal abstract class BaseSheetViewModel<TransitionTargetType>(
 
     internal val activeLinkSession = MutableLiveData(false)
 
+    protected val _linkConfiguration =
```

Comment:
Do we have an initial value for the configuration? If yes, would it make sense to use a `StateFlow`?

Reply:
We don't have an initial value for configuration until the `StripeIntent` is fetched, because we need it to even know if Link should be shown. Once it's fetched and `setupLink` is called, then this will always have a value, even if the activity is destroyed and recreated.
We could use a `StateFlow` instead of `LiveData`, [`SavedStateHandle` offers that too](https://developer.android.com/reference/androidx/lifecycle/SavedStateHandle#getStateFlow(kotlin.String,kotlin.Any)). Here I chose `LiveData` because that's what we use for all other variables in this class.